srcpkgs/python3-ijson/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-ijson'
+pkgname=python3-ijson
+version=3.1.4
+revision=1
+wrksrc="ijson-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools_scm"
+makedepends="yajl-devel python3-devel"
+checkdepends="python3-cffi"
+short_desc="Iterative JSON parser with Pythonic interfaces"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="BSD-3-Clause"
+homepage="https://github.com/ICRAR/ijson"
+distfiles="${PYPI_SITE}/i/ijson/ijson-${version}.tar.gz"
+checksum=1d1003ae3c6115ec9b587d29dd136860a81a23c7626b682e2b5b12c9fd30e4ea
+
+post_patch() {
+	rm ijson/backends/yajl.py
```

Comment:
Please leave a comment about https://github.com/void-linux/void-packages/pull/31263#discussion_r653196518 on top of it.